src/App.js

```diff
@@ -1,41 +1,22 @@
-import React, { useState } from 'react';
+import React from 'react';
 import './App.css';
-import Header from './components/header/header';
-import Footer from './components/footer/footer';
-import Survey from './components/allSurvey/survey';
-import { Surveys } from './components/allSurvey/data'
+import Login from './components/user/login';
+import SurveyApp from './components/allSurvey/surveyApp';
+import { Route, Routes } from 'react-router-dom';
 
 function App() {
 
-  const [tabActive, setTabActive] = useState(0);
-  const setTab = (tabId) => {
-    setTabActive(tabId + 1)
-  }
-
-const App = () => {
   return (
     <div>
-      <Header />
+
       <div>
         <div>
-          {
-            Surveys.map((surveyData, id) => {
-              return (
-                <div key={id}>
-                  <Survey
-                    tabId={id}
-                    title={surveyData.title}
-                    questions={surveyData.questions}
-                    setTab={(id) => setTab(id)}
-                    isActive={tabActive === id}
-                  />
-                </div>
-              )
-            })
-          }
+          <Routes>
+            <Route path="/" element={<Login />} />
```

Comment:
@Dharmikpanchani 
Store this path values in constant file of app which you can create in `src/constant/constant.js`.
Once you store the constants in constant.js, it will help you to use it without remembering the pathname

e.g 
`export const ROOT_PATH = '/'`

Reply:
I created constant component and add in App component.